collections/_catalog/deployment/0e76d324-3d2b-4144-a5cb-ff7d7d4aa9df.md

```diff
@@ -3,7 +3,7 @@ layout: item
 name: checkpublish
 type: Deployment
 compatibility: 
-patternId: MESHERY000
+patternId: 0e76d324-3d2b-4144-a5cb-ff7d7d4aa9df
```

Comment:
```suggestion
patternId: MESHERY000
```

Reply:
Need to discuss solution for this, I didn't want to show this design as the first design on our catalog page